protostar/upgrader/upgrade_remote_checker.py

```diff
@@ -0,0 +1,44 @@
+from dataclasses import dataclass
+
+import requests
+
+from protostar.utils.protostar_directory import VersionManager, VersionType
+
+
+class UpgradeRemoteChecker:
+    """
+    Call a remote endpoint to figure out if the new Protostar version is available.
+    """
+
+    PROTOSTAR_REPO = "https://github.com/software-mansion/protostar"
+
+    @dataclass
+    class Result:
+        latest_release_tag: str
+        latest_version: VersionType
+        is_newer_version_available: bool
```

Comment:
It feels weird that this class compares versions and returns this boolean, but it's a minor thing, don't need to change.

Reply:
Actually I think the same. I'll remove it.